run_build.sh

```diff
@@ -115,12 +115,19 @@ function check_daos_linkage() {
 
 function install_python_deps() {
     echo "Installing python build dependencies"
-    cmd="python3 -m pip install --upgrade pip"
+    cmd="python3 -m pip install --user --upgrade pip"
     echo ${cmd}
     eval ${cmd} || return
     cmd="python3 -m pip install --user --ignore-installed distro scons"
     echo ${cmd}
     eval ${cmd} || return
+    # Hack because scons doesn't propagate the environment
+    cmd="/usr/bin/python3 -m pip install --user --upgrade pip"
+    echo ${cmd}
+    eval ${cmd} || return
+    cmd="/usr/bin/python3 -m pip install --user --ignore-installed pyelftools"
+    echo ${cmd}
+    eval ${cmd} || return
```

Comment:
@ravalsam I built and ran some simple tests, and this seems to work. What's going on here is "distro scons" are installed for python3 from the previous module loads, but pyelftools is specifically installed for /usr/bin/python3 because scons doesn't propagate the environment to meson, which is used by spdk since the recent update. If we come up with a better solution we can replace this one, but it seems to be working for now